### ISSUE TITLE
Breadcrumb - change current page to be a link

### ISSUE THREAD

**Issue by babsdenney** [department-of-veterans-affairs/vets-design-system-documentation#2117]
## Description
Update the current page in the breadcrumbs to be a link.

During the governance review, a 'should' change to the breadcrumbs was proposed to change the current page in the breadcrumbs from text to a link. This will also change the action that happens when the current page link is clicked.

The new design would differ from the USWDS design and something we should propose to USWDS to change. 

## Details
- You can read more about the history and accessibility considerations of this issue in the following ticket.
  - [Breadcrumb - Design update#2068](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/2068)
- You can see the new design for the breadcrumb current page in Sketch.
  - [Breadcrumb design](https://www.sketch.com/s/a52734dd-00d0-44f1-9c9e-ff4016130e5c/p/F30D6063-F45E-4AAE-A2F5-A26C3E5EA5A5/canvas)
- Expected action of current page link when clicked. 
  - When the current page link is clicked, it should target #content the way the skip to content link does, which wouldn't trigger a page reload. 
- Review [Design System Engineering Best Practices document](https://vfs.atlassian.net/wiki/spaces/DST/pages/2176516116/Design+System+Engineering+Best+Practices)

## Tasks
- [ ] Review DST backlog for outstanding issues with this component, if necessary
- [ ] Create web component and add to Storybook
- [ ] Write any necessary tests
- [ ] Add Chromatic link to #[add accessibility ticket number] and request review from an accessibility specialist
- [ ] Contact designer for design review
- [ ] Display the appropriate [maturity scale](https://design.va.gov/about/maturity-scale) option in Storybook (once this feature is available)
    - If this is a new component that has not gone through Staging Review, it should be labeled "Use with Caution: Candidate"
- [ ] Merge component
- [ ] Create a new release of component-library
- [ ] Update component-library dependency in vets-design-system-documentation to get the updated component-docs.json
- [ ] Add analytics set-up to `vets-website` repository. See [guidance here](https://vfs.atlassian.net/wiki/spaces/DST/pages/2079817745/Component+development+process#Analytics%5BinlineExtension%5D).
- [ ] Propose code changes to the USWDS

## Acceptance Criteria
- [ ] Component is written and added to Storybook
- [ ] Component has had accessibility and design reviews
- [ ] Design.va.gov has the latest version of component-library
- [ ] Analytics has been configured for the component in the `vets-website` repo
